### ISSUE TITLE
Feature Request: Add Files to .gitignore

### ISSUE THREAD

**Issue by fpnick** [jreybert/vimagit#146]
Good evening,

Another nice short cut would be if I could place my cursor on a non-tracked file in the Magit Window and add (append) it to the .gitignore file with a single (or two) keystroke.
Thinking this one step further, another option would be another keystroke to add all files with the extension of the file under the cursor to the .gitignore file, i.e. *.ext

I do realize that this might be going beyond what you originally wanted to offer with vimagit though and it's maybe one step too far. 

Fabian.

**Comment by jreybert** [comment]
Hi @fpnick,

this feature actually exists, it came in the very beginning of the project. Just press <kbd>I</kbd> when your cursor is on a file, and it's filename will be appended to gitignore.

It is a very dumb feature, which does not do much. Your second point is interesting, I keep it in mind.

My concern is that you did not find it in the documentation. I feel the doc is quite exhaustive, but not structured well enough.

Please, think about it a few minutes and tell me how you think you could have found this information another way:
- different ordering in the doc
- different naming of the function
- different layout of the doc
- an exhaustive screencast with all the commands (would you watch a more than 5 minutes video of this)
- something else?

Thanks for your involvement in vimagit!

**Comment by fpnick** [comment]
Hi @jreybert ,

now that you say it, I found it in the vim help by just searching for 'ignore'. Which I could've done in the first place - sorry!

What I did yesterday was I searched for that feature on the Github page, where I didn't find it. So my suggestion I guess would be to have an exhaustive list of say all the one key commands (having ALL commands in there might be overkill) on the Github page. Although everyone should really be using the vim docs (and I keep saying that to myself, too), something 'clickable' on the web is still a little more comfortable.

Thank you for developing vimagit! I hardly ever do a commit via the command line any more and I code on a daily basis for work...
